From 348851fee7374140603b9e586dba807ec5058e3b Mon Sep 17 00:00:00 2001
From: Hardik T Shah <hardik.t.shah@intel.com>
Date: Tue, 8 Mar 2016 16:00:37 +0530
Subject: [PATCH 3797/4301] SoundWire: Add place holder for the soundwire bus
 driver header file

commit 0651ef8c75af21886eae8f95d7796788ce28ae1b from
git://git@github.com:01org/linux-intel-4.9.git

This patch creates the place holder file for the SoundWire
bus driver header file.

Change-Id: Ib71bbe10f5988194ee24f3793950a6c079d56aa7
Signed-off-by: Hardik T Shah <hardik.t.shah@intel.com>
---
 include/linux/sdw_bus.h |   31 +++++++++++++++++++++++++++++++
 1 files changed, 31 insertions(+), 0 deletions(-)
 create mode 100644 include/linux/sdw_bus.h

diff --git a/include/linux/sdw_bus.h b/include/linux/sdw_bus.h
new file mode 100644
index 0000000..929fd1d
--- /dev/null
+++ b/include/linux/sdw_bus.h
@@ -0,0 +1,31 @@
+/*
+ *  sdw_bus.h - Definition for SoundWire bus interface.
+ *
+ * This header file refers to the MIPI SoundWire 1.0. The comments try to
+ * follow the same conventions with a capital letter for all standard
+ * definitions such as Master, Slave, Data Port, etc. When possible, the
+ * constant numeric values are kept the same as in the MIPI specifications
+ *
+ *  Copyright (C) 2016 Intel Corp
+ *  Author:  Hardik Shah  <hardik.t.shah@intel.com>
+ *
+ *  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; version 2 of the License.
+ *
+ *  This program is distributed in the hope that it will be useful, but
+ *  WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  General Public License for more details.
+ *
+ * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ *
+ */
+#ifndef _LINUX_SDW_BUS_H
+#define _LINUX_SDW_BUS_H
+
+
+
+#endif /*  _LINUX_SDW_BUS_H */
-- 
1.7.5.4

